benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/HubProtocolBenchmark.cs

```diff
@@ -36,16 +36,16 @@ public void GlobalSetup()
             switch (Input)
             {
                 case Message.NoArguments:
-                    _hubMessage = new InvocationMessage("123", true, "Target", null);
+                    _hubMessage = new InvocationMessage(invocationId: null, target: "Target", argumentBindingException: null);
```

Comment:
Why not add an ctor overload of InvocationMessage that doesn't take the invocationId?